src/linagora.esn.contact/app/addressbook/settings/contact-addressbook-settings.pug

```diff
@@ -1,40 +1,41 @@
-sub-header
-  contact-addressbook-settings-subheader(
```

Comment:
the files associated to this component can be deleted i guess

Reply:
You're right. Updated. Nice catch :+1: .